aries-site/src/examples/components/avatar/AvatarCardExampleDo.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Button } from 'grommet';
+import { ShareRounded } from 'grommet-icons';
+import { Card } from '../../templates';
+
+const product = {
+  icon: '/apache-spark.svg',
```

Comment:
```suggestion
  avatar: '/apache-spark.svg',
```

Reply:
Should this take the same structure as the DataTable example where avatar accepts an object with any valid avatar props?